packages/graph/hash_graph/lib/graph/src/api/rest/utoipa_typedef/mod.rs

```diff
@@ -12,5 +12,5 @@ pub mod subgraph;
 #[serde(rename_all = "camelCase")]
 pub struct EntityIdAndTimestamp {
     pub base_id: EntityId,
-    pub timestamp: Timestamp<TransactionTime>,
+    pub timestamp: Timestamp<ProjectedTime>,
```

Comment:
How does `ProjectedTime` de/serialize? From what I can tell it's zero sized and doesn't have any serde impls? Shouldn't this change the outward facing API?

Reply:
It's deserializing as `Timestamp` (as string without the axis), so the OpenAPI is not affected

Reply:
I think we can type this, yes. As it's always tight to the projected axis it gives a little bit more safety in the subgraph library. @Alfred-Mountfield any objections? 
I wouldn't want to do this in this PR however as this would also require adjusting the subgraph library and most likely this would bubble up to the frontend.

Reply:
It's fair to not bloat the work here out to the TS parts for now. Perhaps add a ticket for adjusting the types

Reply:
[done](https://app.asana.com/0/1202805690238892/1203701392652826/f) _(internal)_